src/api/initialDataTypes.js

```diff
@@ -78,6 +78,13 @@ export type InitialDataRealm = {|
   realm_uri: string,
   realm_video_chat_provider: string,
   realm_waiting_period_threshold: number,
+  /**
```

Comment:
nit: blank line to separate comment from item above (and ditto in `getServerSettings.js`)

Reply:
bump -- same thing in `getServerSettings.js`